app/models/user.rb

```diff
@@ -1,15 +1,17 @@
 class User < ApplicationRecord
- has_many :comments 
- has_many :posts
- has_many :likes
+  validates :email, uniqueness: { allow_blank: true }
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
* Kindly add this missing feature to your project as stated in the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Ask%20for%20confirmation%20of%20email.) as currently, your application doesn't ask for confirmation of email

> Ask for confirmation of email.